bundles/asdi/asdi-projection-change/plugin/ProjectionChangerPlugin.js

```diff
@@ -0,0 +1,101 @@
+Oskari.clazz.define( 'Oskari.projection.change.ProjectionChangerPlugin',
+  function ( config, instance ) {
+    this._instance = instance;
+    this._config = config || {};
+    this._clazz = 'Oskari.projection.change.ProjectionChangerPlugin';
+    this._defaultLocation = 'top right';
+    this._index = 55;
+    this._visible = false;
+    this._templates = {
+      projectionchanger: jQuery('<div class="mapplugin oskari-projection-changer"></div>')
+    };
+    this._loc = instance.getLocalization();
+    this._mobileDefs = {
+      buttons:  {
+          'mobile-projectionchanger': {
+              iconCls: 'mobile-projectionchange',
+              tooltip: '',
+              show: true,
+              callback: function () {
```

Comment:
Is an empty callback required for buttons to work? I think we should fix that if it is.